lk2nd/device/dts/msm8953/msm8953-meizu-m1871.dts

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
We prefer BSD-3-Clause for new submissions, unless this is inherited from somewhere.